pom.xml

```diff
@@ -73,7 +73,7 @@
     <cs.jasypt.version>1.9.2</cs.jasypt.version>
     <cs.trilead.version>1.0.0-build217</cs.trilead.version>
     <cs.ehcache.version>2.6.9</cs.ehcache.version>
-    <cs.gson.version>1.7.2</cs.gson.version>
+    <cs.gson.version>[2.4,)</cs.gson.version>
```

Comment:
at the moment this list is a mess. not all deps are in it and the list is unsorted making it very hardd to maintain so I would prefer an automatic update like above.